src/wp-includes/block-patterns.php

```diff
@@ -406,6 +406,11 @@ function _register_theme_block_patterns() {
  * @return array Block pattern data.
  */
 function _wp_get_block_patterns( WP_Theme $theme ) {
+	// Avoid registering patterns on the frontend.
+	if ( ! is_admin() && ! wp_is_json_request() ) {
```

Comment:
Could you update `@return` document to reflect this as well?

Reply:
I copy-pasted the snippet to the wrong function... It should be in the `_register_theme_block_patterns` function instead of `_wp_get_block_patterns`. Fixed.